samples/dashboards/highcharts-components/chart-constructor-gantt/demo.js

```diff
@@ -0,0 +1,90 @@
+const today = new Date(),
```

Comment:
Maybe lets add another component, so that it is visible, that this is a Dashboards demo, not just Highcharts?